Source/NaturalLanguageClassifierV1/Models/Classification.swift

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright IBM Corporation 2015
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import Freddy
+
+extension NaturalLanguageClassifierV1 {
+    
+    public struct Classification: JSONDecodable {
```

Comment:
In general, I think we should try to match the name of the models provided in the Swagger documentation. It will make it easier for our consumers to reference the documentation and also help us maintain the SDK as the services are updated.

But in this case, the documentation uses `NLClassifierOutputPayload`. That's a terrible name. So nice job renaming it to `Classification` instead!